transport_nantes/topicblog/models.py

```diff
@@ -397,12 +397,18 @@ class Meta:
         abstract = True
 
     class StatusChoices(models.TextChoices):
-        # This is the initial state
+        # Newly created objects derived from SendRecordBase start life
+        # as PENDING.
         PENDING = 'PENDING', "Pending"
-        # Toggle to this state when Amazon SES accepts to send the email
+        # If SES indicates a non-permanent failure, the object enters
+        # state RETRY.  From this state, it should eventually
+        # transition to SENT or FAILED.
+        RETRYING = 'RETRY', 'Retry'
```

Comment:
It's not clear to me what events would trigger this state ?

Reply:
An example might be an informational message that the email can't be delivered but will keep trying.

Reply:
The informational message would come from AWS ? 
https://docs.aws.amazon.com/ses/latest/dg/event-publishing-retrieving-sns-contents.html#event-publishing-retrieving-sns-contents-top-level-json-object

Reply:
Yup, e.g. `DeliveryDelay`.

Reply:
The DeliveryDelay event isn't handled by django-ses
![image](https://user-images.githubusercontent.com/70256364/173794062-862d2236-0606-4d75-b92b-c4766e75c016.png)

Reply:
So we should at some point soon.  That list should surely include all the events AWS documents it might send us.  Wdyt?

Reply:
Oh, I see, it's not us, it's django-ses...

Reply:
It's not urgent, but at some point our unknown notification handler should check for things django-ses didn't.  And maybe we should file an issue / offer a patch to django-ses?

Reply:
SGTM.  In the meantime, maybe we add this enum entry whilst we are thinking of it and understand we can't currently transition into that state?

Reply:
Yes, adding it wont hurt anyway.